### ISSUE TITLE
Investigate a way to make posting to other services easier

### ISSUE THREAD

**Issue by BasThomas** [xing/crep#41]
For example posting to GitHub. Maybe in combination with [Peril](https://github.com/danger/peril)?

Out of scope for `crep` itself I feel, but could a "plugin" like #39.

Also referencing [this](https://github.com/rnystrom/GitHawk/issues/848) issue as the idea was mentioned there!

**Comment by JoeSSS** [comment]
Where can we post this on GitHub? Only thing I can think of, is automatic creation of GitHub issues, as I don't think that crashes info can belong to any open PR.

**Comment by BasThomas** [comment]
Yeah, we'd probably have it create an issue with the newsletter / information.
